### ISSUE TITLE
codemeta for zenodo

### ISSUE THREAD

**Issue by rettinghaus** [codemeta/codemeta#213]
For broader acceptance of codemeta, the community should try to look into zenodo/zenodo#1504.

I think `codemeta.json` is basically ready to supersede the badly documented `zenodo.json`.

**Comment by cboettig** [comment]
@rettinghaus yeah, thanks, we're in touch with Lars and team on this, Zenodo was one of a team of repositories that helped shape the codemeta spec to begin with but they also have a lot on their plate.  I'm hopeful will see this in the works soon as an alternative to the `.zenodo.json` mechanism, which I don't think was ever really meant for mainstream use!

**Comment by sdruskat** [comment]
@cboettig Is there a possibility to get involved when this is proceeding to the next stage? I'd like to investigate potential options to create interfaces with different other metadata formats in Zenodo (CFF obviously, but others as well), so that we can take the next step to automating metadata ingestion and conversion to CodeMeta?

**Comment by cboettig** [comment]
@sdruskat sounds great. Can you say a bit more about what you have in mind?  On the Zenodo integration front I think this is mostly straight-forward if possibly tricky implementation details to parse codemeta / schema.org terms into their internal schema on import, instead of reading from the `.zenodo.json` terms (which I believe is basically the native data format under the hood for them, so this may be basically a translator layer between codemeta and `.zenodo.json`.

**Comment by rettinghaus** [comment]
Any updates own this?
